python/paddle/v2/fluid/profiler.py

```diff
@@ -0,0 +1,74 @@
+import paddle.v2.fluid.core as core
+import subprocess
+
+__all__ = ['CudaProfiler']
+
+NV_FLAGS = [
+    "gpustarttimestamp",
+    "gpuendtimestamp",
+    "gridsize3d",
+    "threadblocksize",
+    "streamid",
+    "enableonstart 0",
+    "conckerneltrace",
+]
+
+
+def nvporf_init(output_file, output_mode=None, flags=None):
```

Comment:
We currently use google style for Python comment.

http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
Done. Move the comments to the refined CudaProfiler.